eth/api.go

```diff
@@ -545,3 +545,27 @@ func (api *PrivateDebugAPI) getModifiedAccounts(startBlock, endBlock *types.Bloc
 	}
 	return dirty, nil
 }
+
+// GetBlockReceipts returns all transaction receipts of the specified block.
+func (api *PrivateDebugAPI) GetBlockReceipts(ctx context.Context, blockHash common.Hash) ([]map[string]interface{}, error) {
+	if block := api.eth.blockchain.GetBlockByHash(blockHash); block != nil {
```

Comment:
If you do the error-checks first, the whole clause can be un-indented. Plus the error messages become more accurate.  
```golang
block := api.eth.blockchain.GetBlockByHash(blockHash); 
if block == nil{
	return nil, errors.New("block not found")
}
receipts := api.api.eth.blockchain.GetReceiptsByHash(blockHash)
if receipts == nil{
	return nil, errors.New("receipts not found")
}
```

Reply:
Updated, please check. thanks